docs/user-stories.md

```diff
@@ -0,0 +1,111 @@
+# GLVD User Stories
+
+# USER-01
+
+As a user of Garden Linux I want to know about security issues in Garden Linux so I can operate my systems responsibly.
```

Comment:
One of the things that I yet not understood is, will this be for ALL Users that GL or just SAP based users?

Reply:
I have intentionally not made this distinction here because I think there is nothing SAP specific about this requirement. I think we can have a separate class of user stories for SAP internal users if we need to, but so far I have not found a case where the needs are so different. Please let me know if you have any.